.gitignore

```diff
@@ -1,4 +1,4 @@
-
+.idea
```

Comment:
let's leave this out of here since the project isn't specific to JetBrains software. you can add a global `.gitignore` in your home directory to avoid including files that are incidentally to the tools you yourself use

Reply:
This is a habit, sorry, I only recently switched to webstorm and I _do_ check the files in for some of my other projects so I don't use a global ignore. Happy to remove.